src/style/GetInvolvedPage.less

```diff
@@ -0,0 +1,73 @@
+.GetInvolvedPage {
+    .mission p {
+        a {
+            text-decoration: none !important;
+            transition: 0.3s;
+            transform: scale(1) rotate(0deg);
+            display: inline-block;
+            &:hover {
+                transform: scale(2) rotate(20deg);
+                transition: 0.3s;
+            }
+        }
+    }
+    .w-md-50 {
```

Comment:
These look like handy utility classes. Promote them to their own top-level file?